app/views/assignments/_assignment_form_options.html.erb

```diff
@@ -7,13 +7,13 @@
   <div class="form-checkbox">
     <label><%= f.radio_button :public_repo, true %> Public</label>
     <%= mega_octicon('repo') %>
-    <span class="note">Anyone can see this repository. You choose who can commit.</span>
+    <span class="note">Anyone can see this repository and it's contents.</span>
```

Comment:
In this context, we're not talking about a repository. We're talking about all student submissions, maybe this:

> Students will submit their projects using public repositories. All submissions will be visible to the world.